projects/microphysics/argo/prog-run-and-eval.yaml

```diff
@@ -0,0 +1,163 @@
+      - name: on_off_flag
+    tolerations:
+    - key: "dedicated"
+      operator: "Equal"
+      value: "med-sim-pool"
+      effect: "NoSchedule"
+    container:
+      image: "us.gcr.io/vcm-ml/prognostic_run:811d4a88b8b8f29d727f2d61f46f96b770c8ab47"
+      imagePullPolicy: Always
+      workingDir: "/fv3net/projects/microphysics"
+      resources:
+        requests:
+          memory: "6Gi"
+          cpu: "7500m"
+        limits:
+          memory: "8Gi"
+          cpu: "7500m"
+      envFrom:
+      - secretRef:
+          name: wandb-andrep
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS 
+        value: /secret/gcp-credentials/key.json
+      volumeMounts:
+      - mountPath: /secret/gcp-credentials
+        name: gcp-key-secret
+      command: ["bash", "-c", "-x", "-e"]
+      args: 
+      - |
+        WANDB_RUN_GROUP="{{inputs.parameters.tag}}"
+        echo "{{inputs.parameters.config}}" > fv3config.yaml
+        python3 scripts/prognostic_run.py \
+          --tag "{{inputs.parameters.tag}}" \
+          --config-path fv3config.yaml \
+          --model "{{inputs.parameters.tf_model}}" \
+          "{{inputs.parameters.on_off_flag}}"
+  - name: piggyback-diags
+    inputs:
+      parameters:
+      - name: tag
+    tolerations:
+    - key: "dedicated"
+      operator: "Equal"
+      value: "med-sim-pool"
+      effect: "NoSchedule"
+    container:
+      image: us.gcr.io/vcm-ml/fv3net:811d4a88b8b8f29d727f2d61f46f96b770c8ab47
+      imagePullPolicy: Always
+      resources:
+        requests:
+          memory: "6Gi"
+          cpu: "3000m"
+        limits:
+          memory: "15Gi"
+          cpu: "3000m"
+      envFrom:
+      - secretRef:
+          name: wandb-andrep
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS 
+        value: /secret/gcp-credentials/key.json
+      volumeMounts:
+      - mountPath: /secret/gcp-credentials
+        name: gcp-key-secret
+      command: ["bash", "-c", "-x", "-e"]
+      args: 
+      - |
+        cd projects/microphysics
+        LS=$(ls ./)
+        python3 scripts/piggy-back.py {{inputs.parameters.tag}}
+  - name: prognostic-diags
+    inputs:
+      parameters:
+      - name: tag
+      - name: baseline_tag
+    tolerations:
+    - key: "dedicated"
+      operator: "Equal"
+      value: "med-sim-pool"
+      effect: "NoSchedule"
+    container:
+      image: us.gcr.io/vcm-ml/fv3net:811d4a88b8b8f29d727f2d61f46f96b770c8ab47
+      imagePullPolicy: Always
+      resources:
+        requests:
+          memory: "6Gi"
+          cpu: "3000m"
+        limits:
+          memory: "15Gi"
+          cpu: "3000m"
+      envFrom:
+      - secretRef:
+          name: wandb-andrep
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS 
+        value: /secret/gcp-credentials/key.json
+      volumeMounts:
+      - mountPath: /secret/gcp-credentials
+        name: gcp-key-secret
+      command: ["bash", "-c", "-x", "-e"]
+      args: 
+      - |
+        cd projects/microphysics
```

Comment:
See my note above about adding `WAND_RUN_GROUP` and better leveraging k8s features here too.